### PR TITLE
add JWT token requirement note

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/DAR/issues/240

Added note about JWT token requirement.

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Configure InfluxDB Enterprise to use LDAP (Lightweight Directory Access Protocol
 - Synchronize InfluxDB and LDAP so each LDAP request doesn't need to be queried
 
 {{% note %}}
-To configure InfluxDB Enterprise to support LDAP, all users must be managed in the remote LDAP service.
+LDAP **requires** JWT authentication. To configure InfluxDB Enterprise to support LDAP, all users must be managed in the remote LDAP service.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md:21]: Is this JWT authentication with InfluxDB, or JWT auth with the LDAP service itself?

Comment (lwandzura) [reply content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md:21]: with InfluxDB

Comment (sanderson) [reply content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md:21]: Gotcha. We should link "JWT authentication" to `/enterprise_influxdb/v1.9/administration/configure/security/authentication/#configure-authentication-using-jwt-tokens`

Comment (lwandzura) [reply content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md:21]: Done!!

**Review by sanderson — APPROVED** [review]
🙌🏻

**Pull request merged**